ltest/src/ltest/util.clj

```diff
@@ -206,3 +206,12 @@
 (defn nano->micros
   [nano]
   (.toMicros TimeUnit/NANOSECONDS nano))
+
+(defn nano->human-readable
+  [nano]
+  (let [len (count (str nano))]
```

Comment:
Comparison should probably be done based on numeric value instead of string value ...

Reply:
Yeah, that's a better approach. 

However, let's get this contribution merged and then create a new ticket to address that feedback.